### PR TITLE
Add transcoder address to job info

### LINKED ISSUE #190: Add transcoder address to job info in SDK
**What's the problem? (required)**

The SDK function `getJob()` currently will not return a transcoder address in the result object if the transcoder did not already submit an on-chain claim to store the transcoder address on-chain. In this case, `getJob()` should call the contract function [electActiveTranscoder](https://github.com/livepeer/protocol/blob/master/contracts/bonding/BondingManager.sol#L651) in order to compute the assigned transcoder address to be returned with the result object.

**What's the current behavior? (required)**

The SDK function `getJob()` currently will not return a transcoder address in the result object if the transcoder did not already submit an on-chain claim to store the transcoder address on-chain.

### PR DESCRIPTION
**Pull request by Randy1Burrell**
**What does this pull request do? Explain your changes. (required)**

The SDK function getJob() currently will not return a transcoder address in the result object if the transcoder did not already submit an on-chain claim to store the transcoder address on-chain. This PR calls the contract function [electActiveTranscoder](https://github.com/livepeer/protocol/blob/master/contracts/bonding/BondingManager.sol#L651) in order to compute the assigned transcoder address to be returned with the result object.

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

Fixes #190 
**Screenshots (optional):**

**Checklist:**

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Review by yondonfu — COMMENTED** [review]
Change looks good. Should we add a test in `index.test.js` to check that the shape of the return object from `getJob()` matches the expected schema?

**Comment by Randy1Burrell** [comment]
@yondonfu good suggestion. I just added the test.

File: packages/sdk/src/index.js [hunk 1]
```diff
@@ -1627,6 +1627,15 @@ export default async function createLivepeerSDK(
      */
     async getJob(id: string): Promise<Job> {
       const x = await JobsManager.getJob(id)
+      // Computer transcoder's address if none was returned
+      if (!x.transcoderAddress) {
```
Comment (yondonfu) [inline packages/sdk/src/index.js]: If the job does not yet have a transcoder address attached to it `x.transcoderAddress` will actually be the null/empty address 0x0000000000000000000000000000000000000000

So we could change this check to `x.transcoderAddress === EMPTY_ADDRESS` (where EMPTY_ADDRESS is a constant already defined within the SDK)

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: packages/sdk/src/index.js (hunk 1 above)
Comment (Randy1Burrell) [reply packages/sdk/src/index.js]: @yondonfu I made the change.

**Review by yondonfu — APPROVED** [review]
🚢

**Pull request merged**